app/services/invoices/calculate_fees_service.rb

```diff
@@ -323,7 +323,8 @@ def refresh_amounts(credit_amount_cents:)
     end
 
     def not_in_finalizing_process?
-      (invoice.draft? || invoice.voided?) && context != :finalize
+      !finalizing_invoice?
+      # (invoice.draft? || invoice.voided?) && context != :finalize
```

Comment:
by the previous logic if we send 
- an invoice that is not in draft status and 
- context != :finalize,
we'd consider it as `is_in_finalizing_process`, because first condition returns false, and operator `&&` would return `false`